arches_lingo/src/arches_lingo/components/scheme/editor/SchemeEditor.vue

```diff
@@ -0,0 +1,64 @@
+<script lang="ts">
+const MAXIMIZE = "maximize";
+const SIDE = "side";
+const CLOSE = "close";
+</script>
+
+<script setup lang="ts">
```

Comment:
these two script tags should be combined

Reply:
I can't; because these are used in defineEmits, I get an eslint check hook error when trying to commit that.  There's a good explanation for that behavior [here](https://stackoverflow.com/questions/69951687/vue-3-defineprops-are-referencing-locally-declared-variables).  Alternatively, I can do an eslint ignore as is mentioned in that post.

Reply:
oh fair. In that case I'd say either export them to `constants.ts` or don't use the variables in `defineEmits`. IMO we should avoid double `<script>` tags when possible

Reply:
reverted to strings, these feel more natural locally defined than in constants.